Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/OnlineTranslation.rst

```diff
@@ -0,0 +1,120 @@
+
+Preconditions
+=============
+
+You need a detailed understanding of how TYPO3 works. You have prefereably worked with TYPO3 for some years as developer, administrator,
+integrator or senior editor. You know your way around in Backend and you are familiar with most of the functionality and features.
+If you are more focused in translation extensions, you’ll need to understand all the parts of the extension before you start translating.
+
+What skills are needed
+======================
+
+You need to be bilingual: Fluent in both English and the language you will translate into. It’ll be hard work if you only have casual knowledge of the target language or English. And we end up with confusing localization.
+
+A good understanding of how language is constructed in terms of nouns, verbs, punctation and grammar in general will be necessary.
+
+How to make (good) translations
+===============================
+
+1. Be true to the source labels you work with. Given that the developer of the code, who made the English text, understands the functionality best, please try to translate the meaning of the sentences.
+
+2. Translate organically, not literally. The structure or your target language is important. English often has a different structure and tone, so you must adapt the equal text but the equivalent. So please don’t replicate but replace.
+
+3. Use the same level of formality. The cultural context can be very different from different languages. What goes in English might be way to informal in your language and vice versa. Try to find a good level of (in)formality and stick to it. And be open to discuss it with your fellow team translators.
+
+4. Look into other localized projects in your language. There are tons of open source projects being translated, also into your language. Be curious and look at how the localization is done – there might be things to learn and adapt.
+
+5. Be consistent. Localization of high quality is characterised by the consistency. Make extensive use of the terms and glossary.
+
+6. Make careful use of machine translation. It’s tempting but dangerous to do a quick translation with one of the common machine translation tools and it can sometimes help you getting a deeper understanding of the meaning of a text.
+But very often a machine translated text breaks all of the rules above, unless you carefully rework the text afterwards.
+
+7. Work together. As in all other aspects of Open Source, things get so much better, if we work together. So, reach out for help if you get stuck. Or offer your knowledge if someone ask.
+Crowdin offers a good platform for cooperating with your team translators, and please join the `Translation Slack channel <https://typo3.slack.com/archives/C032FRT0W>`__.
+
+Translation Styles
+==================
+
+In general and where it makes sense, we follow the style guide from the Documentation Team: `<https://typo3.org/community/teams/content/writing-style-guide/>`__
```

Comment:
This is the style guide from the **content team** (not documentation team). the content style guide is authoritative. 

The documentation team also has a style guide. This is based on style guide from content team, but makes some modifications (e.g. capitalization of titles) where it makes sense for docs.typo3.org and adds some additional points that are only relevant for documentation.